src/mainboard/protectli/vault_cml/bootblock.c

```diff
@@ -19,6 +19,8 @@ void bootblock_mainboard_early_init(void)
 	ite_reg_write(GPIO_DEV, 0x29, 0xc0);
 	/* External CLKIN PCICLK */
 	ite_reg_write(GPIO_DEV, 0x71, 0x08);
+	/* Set WDT timeout to 0 */
+	ite_reg_write(GPIO_DEV, 0x73, 0x00);
```

Comment:
Let's add it to `src/superio/ite/common/early_serial.c: ite_kill_watchdog` function with the register definition.

Reply:
Would have to check if it doesn't break other SuperIOs - although it should be fine since that's also how [the Linux it87_wdt driver](https://github.com/torvalds/linux/blob/e0dccc3b76fb35bb257b4118367a883073d7390e/drivers/watchdog/it87_wdt.c#L206) stops the watchdog

Reply:
Done, please check